external/malmo/Minecraft/launchClient.sh

```diff
@@ -68,8 +68,8 @@ fi
 
 # Finally we can launch the Mod, which will load the config file
 while
-    ./gradlew setupDecompWorkspace
-    ./gradlew build
-    ./gradlew runClient
+    ./gradlew setupDecompWorkspace --stacktrace
```

Comment:
Can you remove the `--stacktrace` options in this file? They are for debugging purposes and we don't need the end-user to see the stack traces.